superset-frontend/src/dashboard/components/menu/PopoverDropdown.jsx

```diff
@@ -89,25 +90,31 @@ class PopoverDropdown extends React.PureComponent {
     const { id, value, options, renderButton, renderOption } = this.props;
     const selected = options.find(opt => opt.value === value);
     return (
-      <DropdownButton
+      <Dropdown
         id={id}
-        bsSize="small"
-        title={renderButton(selected)}
-        className="popover-dropdown"
+        trigger="click"
+        overlayStyle={{ zIndex: 3001 }}
```

Comment:
I'm a little hesitant about random zIndex values re-appearing in the codebase after making some effort to consolidate them. I see a couple paths forward:
• Making a z-index registry in the Theme (painful for now, due to the external repo)
• Exporting a z-index value from the Popover itself, and importing here to reference/increment.

Reply:
I changed to pull `z-index` from the theme for now but I really like the idea of a z-index registry. We should do that in the future.